src/components/progressiveImage/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {View, StyleSheet, Animated} from 'react-native';
+import {AnimatedImage} from 'react-native-ui-lib'; //eslint-disable-line
+
+/**
+ * @description: Image component that loads first a small thumbnail of the images, 
+ *               and fades-in the full-sized image with animation once it's loaded
+ * @extends: Animated.Image 
+ * @example: https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/ProgressiveImageScreen.js
+ */
+class ProgressiveImage extends React.Component {
```

Comment:
Add `static displayName = 'ProgressiveImage ';`
We need for our auto generated docs site.